src/Controls/src/Core/DatePicker/DatePicker.cs

```diff
@@ -60,6 +60,9 @@ public DatePicker()
 		}
 
 		/// <include file="../../docs/Microsoft.Maui.Controls/DatePicker.xml" path="//Member[@MemberName='Date']/Docs/*" />
+#if NET6_0_OR_GREATER
```

Comment:
this is used by Xaml, and Xaml has a built-in conversion from string to DateTime, so this isn't required. please remove

Reply:
same for the other ones